src/connector/src/sink/elasticsearch_opensearch/elasticsearch_opensearch_config.rs

```diff
@@ -85,6 +86,13 @@ pub struct ElasticSearchOpenSearchConfig {
     pub r#type: String,
 }
 
+impl EnforceSecretOnCloud for ElasticSearchOpenSearchConfig {
+    const ENFORCE_SECRET_PROPERTIES_ON_CLOUD: phf::Set<&'static str> = phf::phf_set! {
+        "username",
```

Comment:
Do usernames need to be included? I noticed that clickhouse and others are not included, but this one does